app/views/admin/pages/_form.html.erb

```diff
@@ -38,9 +38,43 @@
     </div>
     <div class="form-group">
       <%= form.label :is_top_level_path %>
-      <%= form.check_box :is_top_level_path %>
-      <p>(Determines if it is accessible by <code>/page-slug</code> vs <code>/page/page-slug</code>) Be careful! ⚠️</p>
+      <%= form.check_box :is_top_level_path, class: "crayons-checkbox" %>
+      <p>Determines if it is accessible by <code>/page-slug</code> vs <code>/page/page-slug</code> Be careful! ⚠️</p>
     </div>
+    <% if @page.errors.count > 0 %>
+      <div id="add-landing-page-link-modal-root">
+        <% @page.errors %>
+      </div>
+    <% end %>
+      <span data-controller="modal" data-modal-root-selector-value="#add-landing-page-link-modal-root" data-modal-content-selector-value="#add-landing-page-link-modal"
+            data-modal-title-value="There's another locked screen..." data-modal-size-value="default">
+        <div class="form-group">
+          <%= form.label :landing_page, "Use as 'Locked Screen'" %>
+          <%= form.check_box :landing_page, class: "crayons-checkbox", data: { action: @page.has_a_landing_page? ? "modal#toggleModal" : "" } %>
+          <p>Determines if this page will be used as a landing page for anonymous viewers.</p>
+        </div>
+        <div class="form-group">
+          <%= form.label :overwrite, "Overwrite" %>
+          <%= form.check_box :overwrite, class: "crayons-checkbox" %>
+        </div>
```

Comment:
I've been trying to hide this checkbox since it is necessary to overwrite the landing page, but repetitive since we already have an "Overwrite current locked screen" button. That said, I have tried a handful of different techniques to hide this `div`/checkbox, such as using `display:none;`, `visibility: hidden;`, and `class="hidden"`, but the issue that I'm running into is that when I add these styles/classes the "Overwrite current locked screen" button becomes disabled. Any thoughts on how to hide this while keeping the intended behavior? 🤔

Reply:
I would suggest switching to a [hidden input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden) as I don't think it's strictly useful to the Admin, or you could do away with it altogether :D

Reply:
Hmm, maybe I'm missing something because I cannot seem to get this to work using the `hidden` `input`! I've also tried removing the checkbox altogether, but by doing that, there is nothing that actually toggles the `overwrite` attribute and I'm unable to `overwrite` the current landing page. 🤔 

The goal is to have this checkbox be hidden so that the feature works as expected without having to include the additional form field to get it to work in that manner. I'm very open to ideas for how to potentially get rid of the checkbox altogether, while still being able to `overwrite` the current landing page as expected, though @rhymes! 😊